### PR TITLE
Added Text-utilities and Lists

### PR DESCRIPTION
**Pull request by sri-rishi**
Text-Utilities :

- Headings
- Gray color text
- Small Text
- Underline text
- Strike-through text
- Center align text

List:

- Standard List 
- Stacked List

### CODE REVIEW

File: Docs/Components_html/lists.html [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+                        
+                    </p>
+                </div>
+
+                <div class="component-display-box font-weight-5">
+                   
+                </div>
+
+                
+            </section>
```
Comment (rahulk31) [inline Docs/Components_html/lists.html]: unnecessary spaces can be removed.

**Comment by rahulk31** [comment]
Overall looks good to me, Ready to Merge.
Though I have a small suggestion, please dont do commits for every small changes.

File: Docs/Components_html/lists.html (hunk 1 above)
Comment (sri-rishi) [reply Docs/Components_html/lists.html]: I don't know what happened here but in here there are components which is not showing here but everywhere else it's showing components even in github code section you will find them.

**Review by Naman-Saxena1 — CHANGES_REQUESTED** [review]
Your carbon code snippet container is scrolling and has very small height, try making its height to atleast fit-content for better readability.

**Comment by sri-rishi** [comment]
> 

done with it

**Pull request merged**